README.md

```diff
@@ -200,6 +200,9 @@ Command flag              | Description
 `-s/--stdin`              | Pipe source from STDIN. This is useful for editor integration.
 `--[no-]color`            | Force color output on or off.
 
+Configuration options are loaded from `.rubocop`, command line switches, and the `RUBOCOP_OPTS` environment variable (listed in
```

Comment:
I'd use a wording like "Default command-line options are loaded from `.rubocop` and `RUBOCOP_OPTS` and are combined with command-line options that are explicitly passed to `rubocop`.". You should probably mention that explicitly passed params are going to have higher precedence.